index.html

```diff
@@ -0,0 +1,794 @@
+          development, from your choice of images to the performance of your web server. In this guide we bring together
+          information that can help you build a fast site, and keep it fast.
+        </p>
+        <button class="explore__and__newsletter-item-button grey-purple-bg" type="button">Explore
+          Performance</button>
+      </article>
+      <div class="newsletter__cat">
+        <div class="newsletter__cat-image">
+          <img src="assets/newsletter.png" alt="">
+        </div>
+        <div class="newsletter__cat-content">
+          <a class="ncc-a" href="https://www.smashingmagazine.com/the-smashing-newsletter/">
+            <h1 class="ncc-h1">Email Newsletter</h1>
+          </a>
+          <p class="ncc-p">
+            Every second Tuesday, we send an email newsletter with useful techniques on front-end and UX. Subscribe and
+            get <strong>Smart Interface Design Checklists PDF</strong> in your inbox.
+          </p>
+        </div>
+        <div class="newsletter__cat-submit">
+          <input class="newsletter__cat-submit-input" type="email" placeholder="Your email ">
+          <button class="newsletter__cat-submit-button" type="button">Meow!</button>
+        </div>
+  
+        <div class="newsletter__cat-last-text">
+          <div class="newsletter__cat-last-text-wrapper">
+            <p>
+              Front-end, UX and design. 2× a month.
+            </p>
+            <p>
+              You can always unsubscribe with just 1 click.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="divider">
+      <a class="divider-a" href="https://www.smashingmagazine.com/guides/">see all guides<i
+          class="fas fa-long-arrow-alt-right divider-i"></i></a>
+    </div>
+  </main>
+  <footer>
+    <div class="footer">
+      <div class="footer__title">
+        <h5>BROWSE ALL SMASHING MAGAZINE TOPICS</h5>
+      </div>
+      <div class="footer__links-list">
+        <ul class="footer__links-list-ul">
+          <li class="footer__links-list-li"><a class="footer__links-list-a" href="#">Accessibility</a></li>
+          <li class="footer__links-list-li"><a class="footer__links-list-a" href="#">Android</a></li>
+          <li class="footer__links-list-li"><a class="footer__links-list-a" href="#">Animation</a></li>
+          <li class="footer__links-list-li"><a class="footer__links-list-a" href="#">Apps</a></li>
+          <li class="footer__links-list-li"><a class="footer__links-list-a" href="#">CSS</a></li>
+          <li class="footer__links-list-li"><a class="footer__links-list-a" href="#">Design</a></li>
+          <li class="footer__links-list-li"><a class="footer__links-list-a" href="#">Design Patterns</a></li>
+          <li class="footer__links-list-li"><a class="footer__links-list-a" href="#">Design Systems</a></li>
+          <li class="footer__links-list-li"><a class="footer__links-list-a" href="#">E-Commerce</a></li>
```

Comment:
- [ ] **MANDATORY** Footer should contain centered links. The result must be something like this:
![image](https://user-images.githubusercontent.com/53256139/87844626-7c028680-c884-11ea-8979-d354c6e05dfe.png)
__*Hint:*__ you will have to modify the css of the `<ul class="footer__links-list-ul">` and the `<li class="footer__links-list-li">` . 
__*Hint2:*__ The `<li class="footer__links-list-li">` needs to have a width that adapts to the lenght of the text and margin for the space between li's

Reply:
Fix footer links.